RealEstate/apps/core/views.py

```diff
@@ -1,7 +1,15 @@
 from django.shortcuts import render
 from django.views.generic import View
+from django.contrib.auth import authenticate, login, logout
+from django.http import HttpResponseRedirect
+from django import forms
+from .models import House, Homebuyer, Couple
 
 
 class HomeView(View):
     def get(self, request, *args, **kwargs):
-        return render(request, 'core/homebuyerHome.html', {})
+        if request.user.is_authenticated():
+          couple = Couple.objects.filter(homebuyer__user=request.user)
+          house = House.objects.filter(couple=couple)
+          return render(request, 'core/homebuyerHome.html', {'couple': couple, 'house': house})
+        return HttpResponseRedirect('/login/')
```

Comment:
Rather than hard coding URLs, lets use django.core.urlresolvers.reverse.  So for example, in urls.py, the login URL is named 'auth_login'.  reverse('auth_login') would then return the correct '/login/' URL.  This lets us configure our URLs all in one spot, and if we decide to change any of them, we change them in one place rather than hunting them all down in the code.